### ISSUE TITLE
support for other scm

### ISSUE THREAD

**Issue by markuman** [conwnet/github1s#35]
Support other scm (_gitlab, gitea_) when selfhosting for an other git platform.

**Comment by softmarshmallow** [comment]
Here's a support for gitlab and bitbucket
https://github.com/bridgedxyz/github.surf

**Comment by xcv58** [comment]
> Here's a support for gitlab and bitbucket
> https://github.com/bridgedxyz/github.surf
> 
> * [github.surf](https://github.surf)
> * [gitlab.surf](https://gitlab.surf)
> * [bitbucket.surf](https://bitbucket.surf)

No, your deployment clearly doesn't support gitlab and bitbucket. It's still reading data from GitHub.

**Comment by markuman** [comment]
It would be necessary that you can point to an other host (e.g. env var) when you deploy it. E.g. when you selfhost gitlab in your company or selfhost a gitea for fun and profit.

**Comment by justinzzc** [comment]
any update plan?
